File: Server/src/ArmorManager.ts
import { AbstractModManager } from "./AbstractModManager"
import { Constants } from "./Constants"
import { Helper } from "./Helper"


export class ArmorManager extends AbstractModManager
{
    protected configName: string = "ArmorConfig"

    protected afterPostDB(): void
    {
        Helper.iterateConfigItems(this.databaseTables, this.config)
        
        console.log(`${Constants.ModTitle}: Armor changes applied!`)
    }
}

File: Server/VAI-NRTP/Managers/ArmorManager.cs
using SPTarkov.DI.Annotations;

namespace VAI.NRTP.Managers;

[Injectable(InjectionType.Singleton)]
public sealed class ArmorManager : AbstractModManager
{
    protected override string ConfigName => "ArmorConfig";

    protected override void AfterPostDb()
    {
        Helper.IterateConfigItems(DatabaseTables, Config);

        Constants.GetLogger().Info($"{Constants.ModTitle}: Armor changes applied!");
    }
}
